crates/release_plz/tests/all/helpers/test_context.rs

```diff
@@ -92,6 +92,14 @@ impl TestContext {
             .assert()
     }
 
+    pub fn disable_changelog_update(&self) {
+        let cargo_toml_path = self.repo_dir().join("Cargo.toml");
+        let mut cargo_toml = LocalManifest::try_new(&cargo_toml_path).unwrap();
+        cargo_toml.data["workspace"]["metadata"]["release-plz"]["workspace"]["changelog_update"] =
+            toml_edit::Item::Value(toml_edit::Value::Boolean(toml_edit::Formatted::new(false)));
+        cargo_toml.write().unwrap();
+    }
```

Comment:
You are definitely on the right path. Can you commit after you do this change?
The test is failing because the Cargo.toml is not committed.

Reply:
Done in 9a3aa94ab43dbadea68a77f550614b697ed5e5b8